### DISCUSSION TITLE
Кнопка "Перемотать полторы минуты"

### DISCUSSION THREAD

**Discussion by Cipher4444** [cawa-93/anime-library#300] [Новые идеи]
Было бы неплохо иметь такую кнопку, чтобы сразу скипать опенинги и эндинги. А то всё время мышкой попадать на нужное место надоедает.

**Comment by cawa-93** [comment]
`L`/`J` -- Перемотка вперёд/назад на 10 секунд (как на youtube). Если при этом удерживать Shift то скорость перемотки удваивается (20 секунд). Ксожелению реализовать кнопку для автоматической перемотки опенинга технически очень сложно (сам бы хотел). А кнопка для перемотки ровно на полторы минуты не очень целесообразно, так как все опенинги разные по длительности: где-то полторы минуты буде мало и нужно будет проматывать ещё, а где-то -- слишком много и нужно будет отматывать назад.
